js/apps/system/_admin/aardvark/APP/react/src/views/collections/addCollection/AddCollectionModal.tsx

```diff
@@ -136,7 +136,7 @@ export const AddCollectionModal = ({
       isSatellite,
       ...extra
     } = values;
-    const clusterOptions = window.App
+    const clusterOptions: Partial<CreateCollectionOptions> = window.App
```

Comment:
I don't think this was intentional here. Let's revert any unnecessary changes in this PR